### PR TITLE
WIP: dockerizing startup (local env)

### PR DESCRIPTION
**Pull request by thxv3n0lvl**
Added docker files, there's an issue while bringing up the containers, this is just an IN PROGRESS (or draft) PR.
The idea behind this PR is to dockerize the local env so contributors don't need to execute all steps manually and to have a starting point to dockerize prod and orchestrate through Kubernetes or something.

Update  (Nov 17, 2020) -
The container name must be used in the configurations files (.env) in order to allow docker configure "everything" for us, for instance:
```
# package/api/.env
# these props were setup in the docker-compose.yml from LiteFarm's root 

DEV_DATABASE=pg-litefarm        
DEV_DATABASE_USER=postgres
DEV_DATABASE_HOST=litefarm-db
DEV_DATABASE_PASSWORD=postgres

TEST_DATABASE=pg-litefarm
TEST_DATABASE_USER=postgres
TEST_DATABASE_HOST=litefarm-db
TEST_DATABASE_PASSWORD=postgres
```
The way to make it work is to place the terminal in the root folder and execute 
`docker-compose up`

More work needed to allow hot-reloading and general testing.

### CODE REVIEW

File: packages/api/scripts/init.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+if [ $1 == 'testing' ]
+then
+  echo "--- Seeding DB with test data ---"
+  for i in `seq 1 10`;
+    do
+      nc -z litefarm-db 5432 && echo Success && break
+      echo -n .
+      sleep 1
+    done
+  # echo Failed waiting for Postgres && exit 1
```
Comment (thxv3n0lvl) [inline packages/api/scripts/init.sh:14]: Need to find out a better way to catch the error here...

**Comment by thxv3n0lvl** [comment]
After syncing up with `develop` I wasn't able to run the env successfully ... might be missing something, if it sounds familiar to you, let me know. 

```
litefarm-web | ./src/containers/Shift/EditShiftOne/index.js
litefarm-web | Module not found: Can't resolve '@svgr/webpack' in '/usr/src/app'
```

**Comment by kikemarquez8** [comment]
@thxv3n0lvl This was a task we addressed this sprint, I took most of your changes, modified the compose file to include storybook and separated testing and dev database, if you have 1 minute to review it that would be awesome https://github.com/LiteFarmOrg/LiteFarm/pull/269

**Comment by kikemarquez8** [comment]
BTW, hot reloading for backend, storybook and webapp was working for me

**Pull request closed without merging**